lib/samples/query_feature_table/query_feature_table_sample.dart

```diff
@@ -123,8 +123,8 @@ class _QueryFeatureTableSampleState extends State<QueryFeatureTableSample>
 
     // Create query parameters and set the where clause.
     final queryParameters = QueryParameters();
-    final stateName = value.trim().toUpperCase();
-    queryParameters.whereClause = "upper(STATE_NAME) LIKE '$stateName'";
+    final stateName = value.trim().toUpperCase().replaceAll("'", "''");
```

Comment:
why do you want to replace it with a pair of single quotes instead of removing it `replaceAll("'", "")`

Reply:
Removing single quotes would certainly fix this issue, because we know that no state names have single quotes. But as a more general solution, we should escape single quotes in case the user intentionally wants to search for something that has single quotes. For example, if they were searching for names, they may want to specify "O'Brian". I realize that's not a concern here, but I think our sample code should do it this way in case people copy our code.

Reply:
Oh, good points.

Reply:
I think we should add a comment here to explain that. Something like: 
`// Replace single quotes with a pair of single quotes to form a valid SQL query.`

Since it's specific to the SQL query - otherwise it might raise the question of why not just use a validator, whereas they are valid, it's just we have to meet the SQL reqs. If that makes any sense.